test/src/parsing.py

```diff
@@ -84,6 +84,7 @@ def test_newline_after_nul(self):
         resp = ('MESSAGE\n'
                 'destination:/exchange/amq.fanout\n'
                 'message-id:Q_/exchange/amq.fanout@@session-(.*)\n'
+                'redelivered:false\n'
```

Comment:
if we have to change all the tests do account for `redelivered`, does this means this will break things to users? If yes, should this go to master instead?